### PR TITLE
Hamza/client db class

### PR DESCRIPTION
**Pull request by hamzafarooq360**
# Summary

Write a quick summary of your changes here. Make sure to provide context for people that may be unfamiliar with what you're working on!

added client db class
	
## Test Plan

How did you test your changes?

## Related Issues

Which issues does this PR resolve/work on?
Closes #

### CODE REVIEW

**Comment by hamzafarooq360** [comment]
@peterghrong please review :)

File: backend/README.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 ## How to run database
-TODO
+Make sure you have [Docker Desktop](https://www.docker.com/get-started/) on your machine.
```
Comment (peterghrong) [inline backend/README.md]: What is this

File: backend/src/entities/ClientEntity.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+// Creates an entity
+// Entity is a class that maps to a database table : https://orkhan.gitbook.io/typeorm/docs/entities
+
+@Entity()
```
Comment (peterghrong) [inline backend/src/entities/ClientEntity.ts:4]: Maybe extend User? Like this: https://github.com/hack4impact-mcgill/MADA-v2/pull/38/files

File: backend/README.md (hunk 1 above)
Comment (hamzafarooq360) [reply backend/README.md:2]: this was something u added when soph and I checked the logs I think

Comment (hamzafarooq360) [reply backend/README.md]: > What is this

this was something u added when soph and I checked the logs I think

**Review by peterghrong — CHANGES_REQUESTED** [review]
Package lock shouldn't change if your package json didnt change.

**Pull request closed without merging**